src/main/java/net/nas/pieces/ColorOfChessPiece.java

```diff
@@ -1,5 +1,18 @@
 package net.nas.pieces;
 
+import java.util.function.Function;
+
 public enum ColorOfChessPiece {
-    WHITE, BLACK, BLANK
+    WHITE(String::toLowerCase),
+    BLACK(String::toUpperCase),
+    BLANK((representationString) -> ".");
+    private final Function<String, String> representationFunction;
```

Comment:
```suggestion
    BLANK((representationString) -> ".");
    
    private final Function<String, String> representationFunction;
```
이렇게 띄워주는 편이 보기 더 좋을 것 같아요.

Reply:
그렇네요! 띄운게 좀 더 보기 괜찮은 것 같아요! 이 부분도 반영하겠습니다!